### DISCUSSION TITLE
Errors while running BIM2SIM on the E3D virtual machine: No module named 'OCC'

### DISCUSSION THREAD

**Discussion by Xingjian-Wang** [BIM2SIM/bim2sim#485] [Q&A] [answered]
Hi there,
I attempted to run the BIM2SIM file on the virtual machine but received an error message indicating that the module is not found. I have already successfully installed the module and can confirm that it is installed using the conda list command. The program runs successfully on other computers. Could you please advise on how to resolve this issue?
![OCC error](https://user-images.githubusercontent.com/72249299/219645279-e438e17c-12b1-4544-9037-afebbffac664.png)
![conda list](https://user-images.githubusercontent.com/72249299/219645291-1a29ee0c-eb22-4745-8106-de8b18ae6e4f.png)

Thanks in advance

Best Regards,
Wang

**Comment by veronikarichter** [comment, marked as answer]
Hi, 
It seems to be an installation issue on your machine.  Please make sure to install pythonocc-core (and all other setup requirements) as described [here](https://ebc.pages.rwth-aachen.de/EBC_all/github_ci/bim2sim/development/docs/installation.html). Maybe you need to check your environment variables, I'd recommend the usual troubleshooting with stackoverflow etc.
